src/backend/sgx/thread.rs

```diff
@@ -57,8 +71,71 @@ impl super::super::Keep for super::Keep {
     }
 }
 
+fn sgx_enarxcall<'a>(enarxcall: &'a mut Payload, data: &'a mut [u8]) -> Option<Item<'a>> {
+    match enarxcall {
+        item::Enarxcall {
+            num,
+            argv: [leaf, subleaf, cpuid_offset, ..],
+            ret,
+        } if *num == item::enarxcall::Number::Cpuid => unsafe {
+            let cpuid_ret = deref_aligned::<CpuidResult>(data, *cpuid_offset, 1).unwrap();
```

Comment:
That's an example of an error I would expect to be propagated up the stack, wrapped by the caller and eventually printed out with a trace (which may or may not happen via `panic`)

Reply:
agreed